src/index.html

```diff
@@ -12,7 +12,40 @@
       href="styles/index.scss"
```

Comment:
Incorrect link reference to the SCSS file. In an HTML document, you should link to a CSS file. Make sure to compile your SCSS to CSS and reference the compiled CSS file instead.

Reply:
From description of  home work 

Write styles in src/styles/main.scss instead of src/style.css.